rclcpp/include/rclcpp/timer.hpp

```diff
@@ -169,6 +185,10 @@ class GenericTimer : public TimerBase
     cancel();
   }
 
+  /// Handles the function to execute when the timer is emit a trigger.
```

Comment:
These are derived functions. You don't need to re-document them. I'm pretty sure, doxygen will carry forward the inherited documentation.